app/main/lib/media_crud.py

```diff
@@ -74,24 +74,28 @@ def delete(task, model):
         if task.get("context", {}) in obj.context and len(obj.context) > 1:
             deleted = drop_context_from_record(obj, task.get("context", {}))
```

Comment:
I think this is just to prevent null explosions that could occur had someone sent it through as empty. If you want me to I can remove that and just let it explode. What would you like to do?